bin/env.sh

```diff
@@ -10,6 +10,9 @@ DATAMINING=$DISTRO_DIR ; export DATAMINING
 if [ -z "$COAT_MAGFIELD_TORUSMAP" ]; then
     export COAT_MAGFIELD_TORUSMAP=Symm_torus_r2501_phi16_z251_24Apr2018.dat
 fi
+if [ -z "$COAT_MAGFIELD_TORUSSECONDARYMAP" ]; then
```

Comment:
Is this env var only used in the build script?  The code still only uses $COAT_MAGFIELD_TORUSMAP, right?